### DISCUSSION TITLE
ModuleNotFoundError: No module named 'spacy.lang.en.tag_map'

### DISCUSSION THREAD

**Discussion by rxm113** [explosion/spaCy#9780] [Help: Installation] [answered]
Unable to run due to this issue. I have installed Spacy using the spacy docs. Any ideas why this may be?

**Comment by adrianeboyd** [comment, marked as answer]
See: https://stackoverflow.com/questions/70172383/modulenotfounderror-no-module-named-spacy-lang-en-tag-map-when-installing-pla
